### ISSUE TITLE
LaunchKit CLI

### ISSUE THREAD

**Issue by danielattilasimon** [liquity/dev#209]
⚠️ **TODO**: should probably be made into an epic and populated with subtasks for each feature.

### What is the LaunchKit CLI?
TL;DR: it shall be a wizard-like command line tool that makes it easy to quickly launch a Liquity frontend. (Comparable to [0x-launch-kit](https://github.com/0xProject/0x-launch-kit)).

### Configuration 🔧 
The following pieces of information will needed by the wizard to configure a new frontend instance.

#### Frontend address
The Ethereum address that shall receive LQTY rewards from Stability Deposits made through this frontend. If omitted, the frontend shall tag Stability Deposits with the zero-address, which will result in Depositors receiving 100% of the LQTY rewards.

When given a non-zero address, the wizard shall check whether the address has been registered as a Liquity frontend, and (at the very least) warn if it hasn't.

❓**TBD**: should the wizard handle frontend registration? If so, what kind of wallets should it support? Should it direct the frontend operator to a web app instead, which would handle the registration? If so, can we serve that web app on liquity.org?

#### Infura API key
Optional API key to enable snappier UI state updates in the frontend via [real-time events](https://geth.ethereum.org/docs/rpc/pubsub) over WebSockets (AKA pub/sub). Infura only allows WebSocket connections when providing an API key. When omitted, the frontend will still be fully functional, but it might take longer for the UI to update after state changes.

⚠️ **TODO**: anything more to be configured?

### Delivery
The LaunchKit CLI should ideally be delivered as a package on [npmjs.com](npmjs.com), which would make it super simple to download and run, e.g.: `npx @liquity/launch-kit`.

Running the CLI and providing it with all the needed configuration should result in a Docker Compose environment that is ready to be started with `docker-compose up`. Updates of the frontend itself shall be continuously delivered as images on [Docker Hub](https://hub.docker.com/u/liquity). Updating to the latest version then is as simple as `docker-compose pull`.

### Questions
**HTTPS**: It is a good idea for frontend operators to be serving the app via HTTPS as opposed to plain HTTP in order to build trust with users. Should we build HTTPS into the LaunchKit (via e.g. letsencrypt or custom TLS certificate) or leave it up to the frontend operator to take care of that e.g. using a reverse proxy?

---

[![inside joke](https://img.youtube.com/vi/wREBD2og5iY/0.jpg)](https://www.youtube.com/watch?v=wREBD2og5iY)
